# Example configuration file

genicam:
  cam_pair_0:
    pair_namespace: "genicam_nadir"
    camera_rgb_sn: "S123231"
    camera_rgb_port: 0
    camera_rgb_calib_url: "adr"
    camera_mono_sn: "S231241"
    camera_mono_port: 1
    camera_rgb_calib_url: "adr"

  cam_pair_1:
    pair_namespace: "genicam_side"
    camera_rgb_sn: "S123231"
    camera_rgb_port: 0
    camera_rgb_calib_url: "adr"
    camera_mono_sn: "S231241"
    camera_mono_port: 1
    camera_rgb_calib_url: "adr"
    
